File: eBarberShop/eBarberShop.Model/Requests/NarudzbeUpdateRequest.cs
namespace eBarberShop.Model.Requests
{
    public class NarudzbeUpdateRequest
    {
        public DateTime DatumNarudzbe { get; set; }
        public decimal UkupanIznos { get; set; }
        public bool Status { get; set; }
        public bool? Otkazano { get; set; }
        public int KorisnikId { get; set; }
    }
}

File: eBarberShop/UI/ebarbershop_desktop/lib/models/narudzbe/narudzbe_update_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'narudzbe_update_request.g.dart';

@JsonSerializable()
class NarudzbeUpdateRequest {
  DateTime datumNarudzbe;
  double ukupanIznos;
  bool status;
  bool? otkazano;
  int korisnikId;

  NarudzbeUpdateRequest(this.datumNarudzbe, this.ukupanIznos, this.status,
      this.otkazano, this.korisnikId);

  factory NarudzbeUpdateRequest.fromJson(Map<String, dynamic> json) =>
      _$NarudzbeUpdateRequestFromJson(json);

  Map<String, dynamic> toJson() => _$NarudzbeUpdateRequestToJson(this);
}
